src/App.js

```diff
@@ -21,33 +23,66 @@ function App() {
   const [task, setTask] = useState();
   const [workDate, setWorkDate] = useState();
 
-  function handleFormSubmit(event) {
-    // event.target = form
-    // form.name
-    setName(event.target.name.value);
-    setEmail(event.target.email.value);
-    setPhoneNumber(event.target.phoneNumber.value);
-    setFormState("submitted");
-    event.target.reset();
-    event.preventDefault();
+  // function handleFormSubmit(event) {
+  //   console.log("First form submitted");
+
+  //   // event.target = form
+  //   // form.name
+  //   setName(event.target.name.value);
+  //   setEmail(event.target.email.value);
+  //   setPhoneNumber(event.target.phoneNumber.value);
+  //   setFormState("submitted");
+  //   event.target.reset();
+  //   event.preventDefault();
+  // }
+
+  // function handleFormSubmit1(event) {
+  //   setSchool(event.target.school.value);
+  //   setTitle(event.target.title.value);
+  //   setDate(event.target.date.value);
+  //   setFormState("submitted1");
+  //   event.target.reset();
+  //   event.preventDefault();
+  // }
+
+  // function handleFormSubmit2(event) {
+  //   setCompany(event.target.company.value);
+  //   setPosition(event.target.position.value);
+  //   setTask(event.target.task.value);
+  //   setWorkDate(event.target.workDate.value);
+  //   setFormState("finished");
+  //   event.target.reset();
+  //   event.preventDefault();
+  // }
+
+  // App
+  // -> Education
+  function updatePersonalData(data) {
+    // Put data into my state (App)
+    setName(data.name);
+    setEmail(data.email);
+    setPhoneNumber(data.phoneNumber);
+    setFormState("education");
   }
-  function handleFormSubmit1(event) {
-    setSchool(event.target.school.value);
-    setTitle(event.target.title.value);
-    setDate(event.target.date.value);
-    setFormState("submitted1");
-    event.target.reset();
-    event.preventDefault();
+
+  function updateEducationData(data) {
+    // Put data into my state (App)
+    setSchool(data.school);
+    setTitle(data.title);
+    setDate(data.date);
+
+    setFormState("work-experience");
   }
-  function handleFormSubmit2(event) {
-    setCompany(event.target.company.value);
-    setPosition(event.target.position.value);
-    setTask(event.target.task.value);
-    setWorkDate(event.target.workDate.value);
+  function updateWorkData(data) {
+    // Put data into my state (App)
+    setCompany(data.company);
+    setPosition(data.position);
+    setTask(data.task);
+    setWorkDate(data.workDate);
+
     setFormState("finished");
-    event.target.reset();
-    event.preventDefault();
   }
+
   if (formState === "finished") {
```

Comment:
How do you know this is a string that says finished?

THis should be a constant or enum type. Try to avoid string comparisons

Reply:
Thats fine for now, when he learns typescript that is no longer a problem with typed strings.